aws-lc-rs/src/rsa/key.rs

```diff
@@ -432,6 +433,16 @@ where
         Ok(pkey)
     }
 
+    /// Builds a `PublicEncryptingKey` from the public key components.
+    ///
+    /// # Errors
+    /// `error::Unspecified` if the key failed to verify.
+    pub fn build_encrypting_key(&self) -> Result<PublicEncryptingKey, Unspecified> {
+        let rsa = self.build_rsa()?;
+
+        PublicEncryptingKey::new(rsa)
+    }
```

Comment:
It might be more idiomatic (more useful?) to implement this through the `TryInto` trait instead of an explicit function?